### DISCUSSION TITLE
User update any value in the batch details, the batch start date is auto updated and the date is reduced by 1 from the existing date.

### DISCUSSION THREAD

**Discussion by kamal-singhal** [Sunbird-Knowlg/Community#171] [Bugs]
Hi team,

We are facing issue that Whenever the user update any value in the course batch details, the batch start date is auto updated and the value is reduced by 1 from the existing date in the DB.
Database details:
server: cassandra
keyspace: sunbird_course
table name : course_batch 

We also checked the batch start date in the another table as content_hierarchy, in this table date of start date showing fine as expected. 
**Can you help us to figure out what is the issue that why start date is auto update whenever user update batch details. On the report the batch start date coming from the course batch table so it's showing wrong date on the report**.

We need your help to debug this issue. for your reference, attached the screen shots of UI & data in the database for for your better understanding. 

![DB_data_course_batch](https://github.com/Sunbird-Knowlg/Community/assets/125978084/90de4864-1d61-40a2-af66-d02e6fd10abe)
![Batch_Update_console](https://github.com/Sunbird-Knowlg/Community/assets/125978084/dd582b24-f0fc-4bf0-a7b8-d8b48cfba1c5)

**Comment by pallakartheekreddy** [comment]
@kamal-singhal batch functionality is part of Lern building block. Can you create this bug in the below discussion forum
https://github.com/orgs/Sunbird-Lern/discussions

@AmiableAnil For your reference

**Reply by kamal-singhal** [reply]
@pallakartheekreddy thanks for this info. pls close the same, I have raised the same in lern BB.
